planetarium/NineChronicles
# add arena season 3 event

### Description

add arena season 3 event

### How to test

please check diff 

- notice
https://github.com/planetarium/NineChronicles/pull/1376/files#diff-cea35fe46de5c0e360e4b048ee36c1350a0c36728dda5190c242d414cd82f9cfR89-R93

- banner
https://github.com/planetarium/NineChronicles/pull/1376/files#diff-fed8a850b091a8cd4284e3c6de06e45f15995a1f32e8dcecf9cb0d3777a4bec7R69-R75

### Related Links

[문서](https://www.notion.so/planetarium/3-8f178c9b72c545babada966feda3cc53)
[아사나](https://app.asana.com/0/958521740385861/1202159194170222/f)

### Required Reviewers

@jaeho0103 , @planetarium/9c-dev 

### Screenshot
![arena_notice](https://user-images.githubusercontent.com/40553495/164585684-249e2df0-546d-4247-a911-d2fd0e479a7c.jpg)
![arena_banner](https://user-images.githubusercontent.com/40553495/164585685-53da649d-cdd3-42b0-b977-8b42b3781fd7.jpg)
